### PR TITLE
2.0.x ikasan 1371 v2

### PR DESCRIPTION
**Pull request by andrzej-majewski**
This chnages only concern IKASAN-1371. No log4j changes

### CODE REVIEW

File: ikasaneip/persistence/src/test/resources/hsqldb-config.xml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        <entry key="hibernate.dialect" value="org.ikasan.persistence.hibernate.IkasanSybaseASE157Dialect"/>
+        <entry key="hibernate.show_sql" value="false"/>
+        <entry key="hibernate.hbm2ddl.auto" value="false"/>
+        <entry key="hibernate.flushMode" value="commit"/>
+    </util:map>
+
+    <bean id="ikasan.ds" name="ikasan.xads ikasan.ds"
+          class="org.springframework.jdbc.datasource.DriverManagerDataSource">
+        <property name="driverClassName" value="com.sybase.jdbc4.jdbc.SybDriver" />
+        <property name="url" value="jdbc:sybase:Tds:svc-esbdb01n:50100/Ikasan01" />
```
Comment (andrzej-majewski) [inline ikasaneip/persistence/src/test/resources/hsqldb-config.xml]: @mick-stewart73 
Opposite to the name this is not hsql-conf.xml given it is refering to Sybase instance. 

svc-esbdb01n host does not resolve publicly.

File: ikasaneip/persistence/src/main/java/org/ikasan/persistence/hibernate/IkasanSybaseASE157Dialect.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package org.ikasan.persistence.hibernate;
+
+import org.hibernate.dialect.SybaseASE157Dialect;
+import org.hibernate.dialect.pagination.LimitHandler;
+import org.hibernate.engine.spi.RowSelection;
+
+/**
+ * Created by Ikasan Development Team on 27/12/2017.
+ */
+public class IkasanSybaseASE157Dialect extends SybaseASE157Dialect
```
Comment (andrzej-majewski) [inline ikasaneip/persistence/src/main/java/org/ikasan/persistence/hibernate/IkasanSybaseASE157Dialect.java]: Please remove this class from Ikasan:
As discussed under:
https://github.com/ikasanEIP/ikasan/pull/410

File: ikasaneip/persistence/src/main/java/org/ikasan/persistence/hibernate/IkasanSybaseASE157LimitHandler.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+import org.hibernate.dialect.pagination.AbstractLimitHandler;
+import org.hibernate.engine.spi.RowSelection;
+
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+/**
+ * Created by Ikasan Development Team on 27/12/2017.
+ */
+public class IkasanSybaseASE157LimitHandler extends AbstractLimitHandler
```
Comment (andrzej-majewski) [inline ikasaneip/persistence/src/main/java/org/ikasan/persistence/hibernate/IkasanSybaseASE157LimitHandler.java]: Please remove this class:
https://github.com/ikasanEIP/ikasan/pull/410

File: ikasaneip/persistence/src/test/resources/hsqldb-config.xml (hunk 1 above)
Comment (andrzej-majewski) [inline ikasaneip/persistence/src/test/resources/hsqldb-config.xml]: Again Sybase does not provide in memory instance.

File: ikasaneip/wiretap/src/test/java/org/ikasan/wiretap/dao/HibernateWiretapDaoTest.java [hunk 4]
```diff
@@ -73,75 +68,89 @@
...
+//
+//			this.wiretapDao.save(event);
+//		}
+//
+//	}
 
 	@Test
 	@DirtiesContext
 	public void test_get_harvestableRecords()
 	{
```
Comment (andrzej-majewski) [inline ikasaneip/wiretap/src/test/java/org/ikasan/wiretap/dao/HibernateWiretapDaoTest.java]: given test has no assertions. only system outs.  Can we revert to previous version which was asserting ?

Comment (mick-stewart73) [reply ikasaneip/wiretap/src/test/java/org/ikasan/wiretap/dao/HibernateWiretapDaoTest.java]: As I have stated many times @andrzej-majewski I simply checked in a bunch of work mid development as I was stepping away from the computer and did not want to risk losing work. This was clearly set up to point to our Sybase database and the unit test used to bootstrap my ability to debug hibernate to work out how the implementation worked against Sybase and in particular how I could extend the dialect. Moreover I did not create a pull request nor ask you to review the work. So these comments, albeit correct and an oversight on my behalf, are somewhat unnecessary, as I never claimed to be dev complete, nor requested a review.

**Pull request merged**